docs/modules/WebDriver.md

```diff
@@ -24,13 +24,13 @@ To run Selenium Server you will need Java and Chrome or Firefox browser installe
 
 ### PhantomJS
 
-PhantomJS is a headless alternative to Selenium Server that implements
+PhantomJS is a [headless browser](https://en.wikipedia.org/wiki/Headless_browser) alternative to Selenium Server that implements
```

Comment:
Uuuuuups!
This documentation generated from `src/Codeception/Module/WebDriver.php`
Edit a docblock of a class to change the documentation. 
On each release docblocks are transformed into markdown and published on site